addons/test_mail/tests/test_message_track.py

```diff
@@ -314,6 +314,31 @@ def test_tracked_compute(self):
             ('partner_email', 'char', 'foo@example.com', 'bar@example.com'),
         ])
 
+    def test_track_binary(self):
```

Comment:
Would add a small docstring explaining what we test (aka: a binary field, don't track value but add an entry, ...)